wagtailio/static/sass/components/_utilities.scss

```diff
@@ -0,0 +1,13 @@
+.u {
+  &-sr-only {
```

Comment:
We've added this in `utilities/_utilities.scss` : )

Reply:
haha nice one! I've removed this template now